From e5def89aa713357379925983b305b641448528bc Mon Sep 17 00:00:00 2001
From: root <jperrier@student.42lyon.fr>
Date: Mon, 15 Apr 2024 11:40:09 +0200
Subject: [PATCH] Add thor_kernel

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index e1bf12891cb0..9f8959003fd3 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 VERSION = 6
 PATCHLEVEL = 9
 SUBLEVEL = 0
-EXTRAVERSION = -rc3
+EXTRAVERSION = -rc3-thor_kernel
 NAME = Hurr durr I'ma ninja sloth
 
 # *DOCUMENTATION*
-- 
2.43.0

